File: Java/프로그래머스/0/181938. 두 수의 연산값 비교하기/두 수의 연산값 비교하기.java
class Solution {
    public int solution(int a, int b) {
        int answer = 0;
        String a1=String.valueOf(a);
        String a2=String.valueOf(b);
        for(int i=0; i<a2.length();i++){
            a1+=a2.charAt(i);
        }
        int a3 = Integer.parseInt(a1);
        int b1= 2*a*b;
        if(b1>a3) answer = b1;
        else if(a3>b1 || a3==b1) answer=a3;
        return answer;
    }
}

File: 프로그래머스/0/181938. 두 수의 연산값 비교하기/두 수의 연산값 비교하기.js
function solution(a, b) {
    var answer = 0;
    var a1=String(a);
    var a2=String(b);
    for(var i=0; i<a2.length;i++){
            a1+=a2.charAt(i);
        }
        var a3 = Number(a1);
        var b1= 2*a*b;
        if(b1>a3) answer = b1;
        else if(a3>b1 || a3==b1) answer=a3;
    return answer;
}